.tool-versions

```diff
@@ -1,6 +1,7 @@
 act 0.2.62
 actionlint 1.6.27
 bosh 7.5.6
+credhub 2.9.30
```

Comment:
![Screenshot 2024-05-24 at 11 56 08](https://github.com/cloudfoundry/app-autoscaler-release/assets/317403/7be87814-efd4-4621-9412-af7d13cc5ae2)

seems like it is not finding the version. I will commit it as it is as it will still add a credhub dependency.

Reply:
`2.9.30` is not yet available on nixhub.io but `2.9.29` is: https://www.nixhub.io/packages/credhub-cli

I would simply downgrade one minor version to `2.9.29` : D

Reply:
yes @geigerj0 you are right, I don't know where I got the other number from. See 5b17d9637

Reply:
nice, thanks